agreement/security/agreement_security.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo noupdate="1">
+    <record id="agreement_rule" model="ir.rule">
+        <field name="name">Agreement multi-company</field>
+        <field name="model_id" ref="model_agreement" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

Comment:
@newtratip Please change domain like as v 13.0 in my PR (https://github.com/OCA/contract/pull/628/files#diff-330aba9b038127959c7003d614d9c6985fba69f59eb728ac4594dd2872f5e812R12)
In v14.0 noupdate_changes.xml is not needed.

Reply:
yes, changed domain Agreement multi-company rule and remove migrations folder.